### PR TITLE
10th sender

### PR DESCRIPTION
**Pull request by HumaidALKhathami**

### CODE REVIEW

File: .idea/homeworkSolution [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    }
+}
+
+class Triangle() : Shape(){                          // declaring class with name Triangle inherited from Shape
+    val height = 15.4                                // defining a value of height and setting it to 15.4
+    val base = 13.4                                  // defining a value of base and setting it to 13.4
+
+    override fun computeArea():Double{               // overriding the function computeArea
+        return ((height * base)/2)                   // return value
+    }
```
Comment (shuruqt1000) [inline .idea/homeworkSolution:40]: why use override ? why is it important

**Review by shuruqt1000 — CHANGES_REQUESTED** [review]
your comments are good but I need more explaining your code like why did we inherit ? what's override ? what values has been passed and to where ? why is it important and how dose it affect the rest of the code ?

File: .idea/homeworkSolution (hunk 1 above)
Comment (HumaidALKhathami) [reply .idea/homeworkSolution:40]: we use override to change the function which was inherited from the superclass. the importance of overriding is shown in the homework, we need the same function to do different things depending on the class which was overridden in .

**Comment by shuruqt1000** [comment]
great useful comments 
completed

**Pull request closed without merging**